site/en/docs/privacy-sandbox/feedback/report-2022-q3.md

```diff
@@ -0,0 +1,1372 @@
+surfacing through Google’s internal teams and public forms.
+
+More specifically, meeting minutes for web standard bodies meetings were
+reviewed and, for direct feedback, Google’s records of 1:1 stakeholder meetings,
+emails received by individual engineers, the API mailing list, and the public
+feedback form were considered. Google then coordinated between the teams
+involved in these various outreach activities to determine the relative
+prevalence of the themes emerging in relation to each API.
+
+The explanations of Chrome’s responses to feedback were developed from published
+FAQs, actual responses made to issues raised by stakeholders, and determining a
+position specifically for the purposes of this public reporting exercise.
+Reflecting the current focus of development and testing, questions and feedback
+were received in particular with respect to Topics, Fledge and Attribution
+Reporting APIs. 
+
+Feedback received after the end of the current reporting period may not yet have
+a considered Chrome response.
+
+{% Details %} {% DetailsSummary %}
+
+**Glossary of acronyms**
+
+{% endDetailsSummary %}
+
+<dl>
+<dt>W3C</dt>    <dd><a href="https://www.w3.org/">World Wide Web Consortium</a></dd>
+<dt>PatCG</dt>  <dd><a href="https://www.w3.org/community/patcg/">Private Advertising Technology Community Group</a></dd>
+<dt>IETF</dt>   <dd><a href="https://www.ietf.org/">Internet Engineering Task Force</a></dd>
+<dt>DSP</dt>    <dd>Demand-side Platform</dd>
+<dt>SSP</dt>    <dd>Supply-side Platform</dd>
+<dt>OT</dt>     <dd><a href="/blog/origin-trials/">Origin Trial</a></dd>
+<dt>UA</dt>     <dd><a href="/docs/privacy-sandbox/user-agent/">User Agent string</a></dd>
+<dt>UA-CH</dt>  <dd><a href="/docs/privacy-sandbox/user-agent/">User-Agent Client Hints</a></dd>
+<dt>IP</dt>     <dd>Internet Protocol address</dd>
+<dt>WIPB</dt>   <dd><a href="/docs/privacy-sandbox/gnatcatcher/">Willful IP Blindness</a></dd>
+<dt>IAB</dt>    <dd><a href="https://www.iab.com/">Interactive Advertising Bureau</a></dd>
+<dt>openRTB</dt><dd><a href="https://iabtechlab.com/standards/openrtb/#:~:text=OpenRTB%20is%20the%20communication%20protocol,in%20the%20digital%20advertising%20industry.">Real-time bidding</a></dd>
+<dt>CHIPS</dt>  <dd><a href="/docs/privacy-sandbox/chips/">Cookies Having Independent Partitioned State</a></dd>
+<dt>FPS</dt>    <dd><a href="/docs/privacy-sandbox/first-party-sets/">First Party Sets</a></dd>
+<dt>FedCM</dt>  <dd><a href="/docs/privacy-sandbox/fedcm/">Federated Credential Management</a></dd>
+<dt>IDP</dt>    <dd>Identity Provider</dd>
+</dl>
+
+{% endDetails %}
+
+## General feedback, no specific API/Technology
+
+<table>
+  <tr>
+   <td><strong>Feedback Theme </strong>
+   </td>
+   <td><strong>Summary</strong>
+   </td>
+   <td><strong>Chrome Response</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>(Also reported in Q2)
+<p>
+Usefulness for different types of stakeholders
+   </td>
```

Comment:
```suggestion
   <td>
Usefulness for different types of stakeholders
<p>
(Also reported in Q2)
   </td>
```

Reply:
Tinie tiny nit: I think it would read better like this, make even add a `<small>` tag around the parenthesis. It's already an overwhelming information packed table, some visual hierarchy could be nice.